src/overlays/actors/ovl_En_Hg/z_en_hg.c

```diff
@@ -147,16 +164,16 @@ void EnHg_Destroy(Actor* thisx, PlayState* play) {
     Collider_DestroyCylinder(play, &this->collider);
 }
 
-void func_80BCF354(EnHg* this) {
-    Actor_ChangeAnimationByInfo(&this->skelAnime, sAnimationInfo, 0);
-    this->actionFunc = func_80BCF398;
+void EnHg_SetupIdle(EnHg* this) {
```

Comment:
I think `Wait` better describes this situation than `Idle` since it's explicitly waiting for Player to approach.